### DISCUSSION TITLE
Headers X-PO-... estão funcionando correatamente?

### DISCUSSION THREAD

**Discussion by madjer** [po-ui/po-angular#1364] [Angular]
Estou usando os headers X-PO-No-Error e X-PO-No-Message, quando passa pelo interceptor esses headers são removidos conforme esperado, mas continua apresentando a notificação quando retorna 4xx e 5xx, alguém com o mesmo problema?

**Comment by alinelariguet** [comment]
Você consegue simular pelo stackblitz? https://stackblitz.com/edit/po-ui
